### PR TITLE
Adding Latent Space Interpretation to Discussion

### PR DESCRIPTION
**Pull request by gwaygenomics**
We previously discussed adding this section to the discussion - I was reminded by @cgreene in [this comment](https://github.com/greenelab/deep-review/pull/567#discussion_r156947642).

Happy to consider any suggested changes!

### CODE REVIEW

File: content/06.discussion.md [hunk 1]
```diff
@@ -269,13 +269,39 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
+image and text based applications [@tag:Radford_dcgan; @tag:Word2Vec], but
+applications to genomic and biomedical domains are increasing. For example, Way
+and Greene train a variational autoencoder (VAE) on gene expression from The
+Cancer Genome Atlas (TCGA) and use latent space arithmetic to rapidly isolate
+and interpret gene expression features descriptive of two high grade serous
+ovarian cancer subtypes [@tag:WayGreene2017_tybalt]. The most differentiating
+VAE features were representative of biological processes that are known to
+distinguish the subtypes. Latent space arithmetic with features derived using
+other compression algorithms were not as informative in this context
+[@tag:WayGreene2017_eval]. Furthermore, embedding biomedical text into lower
```
Comment (agitter) [inline content/06.discussion.md]: To tie this to the treatment section, how about adding this after `in this context [@tag:WayGreene2017_eval]`:

> Latent spaces can also support continuous optimization approaches that cannot be directly applied to the original discrete representations [@tag:Gomezb2016_automatic].

Or something along those lines.  This paper uses an autoencoder to map from a chemical structure represented as a string to a continuous space (and back to strings).

Comment (gwaygenomics) [reply content/06.discussion.md]: Great! Yes, this would tie in nicely. I will add in the next commit

**Comment by gwaygenomics** [comment]
Thanks for the suggestion @agitter - I added a reference to the paper but focused a bit more on the application of the latent space.

File: content/06.discussion.md [hunk 2]
```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
 
-Finally, it is sometimes possible to train the model to provide justifications
+It is sometimes possible to train the model to provide justifications
 for its predictions. Lei et al. [@tag:Lei2016_rationalizing] used a generator to
 identify "rationales", which are short and coherent pieces of the input text
 that produce similar results to the whole input when passed through an encoder.
 The authors applied their approach to a sentiment analysis task and obtained
 substantially superior results compared to an attention-based method.
 
+Lastly, interpretation of embedded or latent space features learned through
```
Comment (dhimmel) [inline content/06.discussion.md]: Can we use one sentence per line for new paragraphs?

Comment (gwaygenomics) [reply content/06.discussion.md]: Sure. I was doing this originally, but the rest of the discussion does not have this format. I'll update in a new commit soon

File: content/06.discussion.md [hunk 3]
```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
+[@tag:WayGreene2017_eval]. Embedding discrete chemical structures with
+autoencoders and interpreting the learned continuous representations with latent
+space arithmetic has also facilitated the discovery of drug compounds
+[@tag:Gomezb2016_automatic]. Furthermore, embedding biomedical text
+into lower dimensional latent spaces have improved state of the art name entity
+recognition in a variety of tasks including annotating clinical abbreviations,
+genes, cell lines, and drug names [@doi:10.3390/info6040848;
+@doi:10.1155/2014/240403; @doi:10.18653/v1/w15-3822; @doi:10.18653/v1/w15-3810].
+Other approaches have interpolated through latent space embeddings learned by
+generative adversarial networks (GAN) to interpret unobserved intermediate
```
Comment (agitter) [inline content/06.discussion.md]: `GANs` has been defined at the end of the treat section

File: content/06.discussion.md [hunk 4]
```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
+and Greene train a variational autoencoder (VAE) on gene expression from The
+Cancer Genome Atlas (TCGA) and use latent space arithmetic to rapidly isolate
+and interpret gene expression features descriptive of two high grade serous
+ovarian cancer subtypes [@tag:WayGreene2017_tybalt]. The most differentiating
+VAE features were representative of biological processes that are known to
+distinguish the subtypes. Latent space arithmetic with features derived using
+other compression algorithms were not as informative in this context
+[@tag:WayGreene2017_eval]. Embedding discrete chemical structures with
+autoencoders and interpreting the learned continuous representations with latent
+space arithmetic has also facilitated the discovery of drug compounds
```
[the remark this answers is not in the document]
Comment (gwaygenomics) [reply content/06.discussion.md]: :+1:

File: content/06.discussion.md [hunk 5]
```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
+[@tag:Gomezb2016_automatic]. Furthermore, embedding biomedical text
+into lower dimensional latent spaces have improved state of the art name entity
```
[the remark this answers is not in the document]
Comment (gwaygenomics) [reply content/06.discussion.md]: :+1:

File: content/06.discussion.md [hunk 6]
```diff
@@ -269,13 +269,42 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
+recognition in a variety of tasks including annotating clinical abbreviations,
+genes, cell lines, and drug names [@doi:10.3390/info6040848;
+@doi:10.1155/2014/240403; @doi:10.18653/v1/w15-3822; @doi:10.18653/v1/w15-3810].
```
[the remark this answers is not in the document]
Comment (gwaygenomics) [reply content/06.discussion.md]: I pulled these citations from #567, specifically: https://github.com/greenelab/deep-review/pull/567#discussion_r156947642

File: content/06.discussion.md (hunk 3 above)
Comment (gwaygenomics) [reply content/06.discussion.md]: :+1:

File: content/06.discussion.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (agitter) [reply content/06.discussion.md]: Okay, I suppose that isn't actually a problem though.  No need to change anything.

File: content/06.discussion.md [hunk 7]
```diff
@@ -269,13 +269,39 @@ comparable performance. Towards this end, Che et al. [@tag:Che2015_distill] used
...
 
-Finally, it is sometimes possible to train the model to provide justifications
+It is sometimes possible to train the model to provide justifications
 for its predictions. Lei et al. [@tag:Lei2016_rationalizing] used a generator to
 identify "rationales", which are short and coherent pieces of the input text
 that produce similar results to the whole input when passed through an encoder.
 The authors applied their approach to a sentiment analysis task and obtained
 substantially superior results compared to an attention-based method.
 
+Lastly, interpretation of embedded or latent space features learned through
```
[the remark this answers is not in the document]
Comment (gwaygenomics) [reply content/06.discussion.md]: I decided to add it as a new H4 header. I agree it fits better there - we can move it around too if we feel there's a better spot

**Comment by gwaygenomics** [comment]
Thanks for the comments @agitter - I believe I addressed them in my most recent commits.

@dhimmel - I was able to add to Table 1 with no apparent issue. Does it look ok on your end?

**Comment by dhimmel** [comment]
> Does it look ok on your end?

Yep, GitHub detects no conflicts.

**Comment by agitter** [comment]
Looks good.  I'm going to make on minor commit to change to sentence case in the new header and the merge.

**Pull request merged**